docs/reference-guides/block-api/block-metadata.md

```diff
@@ -51,9 +51,9 @@ Starting in WordPress 5.8 release, we encourage using the `block.json` metadata
 	],
 	"editorScript": "file:./index.js",
 	"script": "file:./script.js",
-	"viewScript": "file:./view.js",
+	"viewScript": [ "file:./view.js", "example-shared-view-script" ],
```

Comment:
These are confusing for me. Can we use an array of `file:./` references here or do they have to be registered elsewhere? I think we need to add some text somewhere that explains this.

Reply:
`example-shared-view-script` needs to be registered elsewhere. This covers the case when you want to use the same script/style with multiple blocks.

Reply:
I don't think this is a blocker, maybe we just need some examples around both cases.

Reply:
Let me check if we can add some hints with links to the related section as a follow-up.

Reply:
Follow-up ready: https://github.com/WordPress/gutenberg/pull/44199.